marshmallow/fields.py

```diff
@@ -371,7 +374,7 @@ class Field(FieldABC):
 
     def __init__(self, default=None, attribute=None, load_from=None, error=None,
                  validate=None, required=False, allow_none=False, load_only=False,
-                 dump_only=False, **metadata):
+                 dump_only=False, missing=None, **metadata):
```

Comment:
`None` should be a valid value for `missing`. Perhaps you can make the default `null` here to allow `None` to be used as a value for missing data.